packages/client/src/v2-events/features/drafts/useDrafts.ts

```diff
@@ -103,6 +103,7 @@ setMutationDefaults(trpcOptionsProxy.event.draft.create, {
       action: {
         createdAt: new Date().toISOString(),
         createdBy: '@todo',
+        creator: 'user',
```

Comment:
I think we could use the existing `TokenUserType` enumeration as this type everywhere:
https://github.com/opencrvs/opencrvs-core/blob/develop/packages/commons/src/authentication.ts#L172

Also I would rename `creator` to something along the lines of `createdByUserType`